### PR TITLE
SG-29851 Updating supported OS versions

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/666051b25ad1589e3be09df5e29f0f7cee6c4147/index.html)

**Comment by shaynacohen** [comment]
Screen Shot 2023-02-08 at 1 31 24 PM

@julien-lang OK for me to publish this update?

File: docs/en/reference/pipeline-integrations/toolkit-engines-supported-os-versions.md [hunk 1]
```diff
@@ -11,6 +11,6 @@ Here is a list of Toolkit engine supported operating systems.
 
 ----------
 
-| Windows | macOS | CentOS |
+| Windows | macOS | Rocky Linux |
 |:-------:|:-----:|:------:|
-|   10    |10.14+ <br>**Note:** M1 is not supported yet, [learn more here](https://community.shotgridsoftware.com/t/13606).    |  7.4   |
\ No newline at end of file
+|   10    |10.14+ <br>**Note:** M1 is not supported yet, [learn more here](https://community.shotgridsoftware.com/t/13606).    |  8  |
```
Comment (julien-lang) [inline docs/en/reference/pipeline-integrations/toolkit-engines-supported-os-versions.md:16]: We still support CentOS 7. We just introduce support of Rocky 8 so we support them both at the moment.
We can have two columns, one for CentOS and one for Rocky if we want. Or only one "Linux" column and list Centos 7.4, Rocky 8.5 on the "version" line

Comment (shaynacohen) [reply docs/en/reference/pipeline-integrations/toolkit-engines-supported-os-versions.md:16]: Thanks @julien-lang ! I'll update...

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/f550fe1f6e43f345b08c95760c5a0a73b2319a26/index.html)

**Review by julien-lang — APPROVED** [review]
LGTM!

**Pull request merged**